app/models/vm_or_template.rb

```diff
@@ -1157,35 +1157,44 @@ def self.post_refresh_ems(ems_id, update_start_time)
     ems = ExtManagementSystem.find(ems_id)
 
     # Collect the newly added VMs
```

Comment:
I think this comment is no longer necessary...it's pretty evident from the variable name.